doc/news/changes/minor/20250924Gassmoeller

```diff
@@ -0,0 +1,9 @@
+Changed: The TimerOutput class now includes run time spent in currently active
+subsections in its reports. Previously it would only report time spent in
+sections that had already been left. Additionally, CPU time spent in sections
+is now reported as the sum over all processes in the given MPI communicator.
+Previously it would only report CPU time on the current process, which would
+mean the time spent in sections would not add up to the total CPU time (which
+is reported as sum over all MPI processes).
```

Comment:
The parenthetical remark seems to apply to the wrong part of the sentence, namely to "the time spent in sections" not the following "the total CPU time". Can you rephrase?